<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTextEdit" name="textEdit_IP"/>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_con">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_dis">
              <property name="text">
               <string>Disconnect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Mínimo</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSlider" name="horizontalSlider_min">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLCDNumber" name="lcdNumber_min">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QSlider" name="horizontalSlider_max">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLCDNumber" name="lcdNumber_max">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Máximo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Timing (s)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_timings">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_start">
              <property name="text">
               <string>START</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_stop">
              <property name="text">
               <string>STOP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_status">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QTextBrowser" name="textBrowser_dados"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>507</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_min</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_max</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_timings</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
